plotting.py

```diff
@@ -563,21 +498,21 @@ def plot_timehistory_at_height(datasets,
             axi.text(-0.14,1.0,'('+chr(i+97)+')',transform=axi.transAxes,size=16)
 
     # Add legend if more than one entry
-    if (stack_by=='datasets' and Ndatasets>1) or (stack_by=='heights' and Nheights>1):
+    if (stack_by_datasets and ndatasets>1) or (not stack_by_datasets and nheights>1):
         leg = axv[ncols-1].legend(loc='upper left',bbox_to_anchor=(1.05,1.0),fontsize=16)
 
     return fig, ax
 
 
 def plot_profile(datasets,
-                 fields,
-                 times,
+                 fields=None,
+                 times=None,
                  fig=None,ax=None,
                  fieldlimits={},
                  heightlimits=None,
                  fieldlabels={},
-                 colormap=None,
-                 stack_by=None,
+                 cmap=None,
+                 stack_by_datasets=None,
```

Comment:
If this is a bool, then why is the default None?

Reply:
By default, you want the function to figure out by itself if it needs to stack by datasets or not based on the number of datasets and the number of heights/times (`plot_timehistory_at_height` or `plot_profile`). So unless the user specifies `stack_by_datasets` to be `True` or `False`, you set it to `None` to signal that is has not been specified. Would it be more clear if I improve the docstring?

Reply:
Yes, please. And to clarify, "stacking by datasets" means each dataset has its own row (and columns correspond to fields), right?

Reply:
"stacking by datasets" means you are plotting ("stacking") multiple datasets in the same subfigure, and different times/heights are plotted in different subplots. Alternative, if `stack_by_datasets=False`, you will stack different times/heights in the same subfigure and have a subfigure per dataset.

Reply:
Okay, stacked means what is plotted in the same subfigure, got it.